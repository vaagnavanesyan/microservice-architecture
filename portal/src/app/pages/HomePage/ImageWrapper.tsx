import * as React from 'react';
import styled from 'styled-components/macro';

export function ImageWrapper({ src }) {
  return (
    <Wrapper>
      <Image src={`${process.env.PUBLIC_URL}/${src}`} alt="Logo" />
    </Wrapper>
  );
}

const Image = styled.img`
  width: 100%;
`;

const Wrapper = styled.main`
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  min-height: 320px;
`;
