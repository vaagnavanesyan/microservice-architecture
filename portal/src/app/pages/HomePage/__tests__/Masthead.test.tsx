import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import { ImageWrapper } from '../ImageWrapper';

const shallowRenderer = createRenderer();

describe('<Masthead />', () => {
  it('should render and match the snapshot', () => {
    shallowRenderer.render(<ImageWrapper />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
