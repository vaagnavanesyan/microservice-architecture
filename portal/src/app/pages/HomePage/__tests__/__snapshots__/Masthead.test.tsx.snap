// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Masthead /> should render and match the snapshot 1`] = `
<Masthead__Wrapper>
  <Masthead__Image
    alt=""
    src="/facialrecognition.png"
  />
</Masthead__Wrapper>
`;
